src/main/java/nl/tudelft/scrumbledore/keybinding/Keybinding.java

```diff
@@ -0,0 +1,95 @@
+package nl.tudelft.scrumbledore.keybinding;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javafx.scene.input.KeyCode;
+import nl.tudelft.scrumbledore.Constants;
+import nl.tudelft.scrumbledore.level.PlayerAction;
+
+/**
+ * Represents a Keybinding in the form of a key value pair.
+ * 
+ * @author Jeroen Meijer
+ */
+public class Keybinding {
+  private Map<KeyCode, PlayerAction> keyMap;
+  private Map<PlayerAction, KeyCode> actionMap;
```

Comment:
This is to make the key action pair unique and to be able to get an action by a key and a key by an action.